mkdocs.yml

```diff
@@ -1,124 +1,207 @@
-        - Change Logging: 'additional-features/change-logging.md'
-        - Computed Fields: 'models/extras/computedfield.md'
-        - Context Data: 'additional-features/config-contexts.md'
-        - Custom Fields: 'models/extras/customfield.md'
-        - Custom Links: 'models/extras/customlink.md'
-        - Dynamic Groups: 'models/extras/dynamicgroup.md'
-        - Export Templates: 'models/extras/exporttemplate.md'
-        - Git Repositories: 'models/extras/gitrepository.md'
-        - Health Check: 'additional-features/healthcheck.md'
-        - Jobs: 'additional-features/jobs.md'
-        - Job Hooks: 'models/extras/jobhook.md'
-        - Job Scheduling and Approvals: 'additional-features/job-scheduling-and-approvals.md'
-        - NAPALM: 'additional-features/napalm.md'
-        - Notes: 'models/extras/note.md'
-        - Prometheus Metrics: 'additional-features/prometheus-metrics.md'
-        - Relationships: 'models/extras/relationship.md'
-        - Statuses: 'models/extras/status.md'
-        - Tags: 'models/extras/tag.md'
-        - Template Filters: 'additional-features/template-filters.md'
-        - Webhooks: 'models/extras/webhook.md'
-    - Plugins:
-        - Using Plugins: 'plugins/index.md'
-        - Developing Plugins: 'plugins/development.md'
-        - Porting NetBox Plugins to Nautobot: 'plugins/porting-from-netbox.md'
-    - Administration:
-        - Nautobot Server: 'administration/nautobot-server.md'
-        - Nautobot Shell: 'administration/nautobot-shell.md'
-        - Permissions: 'administration/permissions.md'
-        - Replicating Nautobot: 'administration/replicating-nautobot.md'
-    - REST API:
-        - Overview: 'rest-api/overview.md'
-        - Filtering: 'rest-api/filtering.md'
-        - Authentication: 'rest-api/authentication.md'
-    - GraphQL API:
-        - Overview: 'additional-features/graphql.md'
-    - Development:
-        - Introduction: 'development/index.md'
-        - Getting Started: 'development/getting-started.md'
-        - Advanced Docker Compose Usage: 'development/docker-compose-advanced-use-cases.md'
-        - Best Practices: 'development/best-practices.md'
-        - Style Guide: 'development/style-guide.md'
-        - Extending Models: 'development/extending-models.md'
-        - Application Registry: 'development/application-registry.md'
-        - Page Templates: 'development/templates.md'
-        - Navigation Menu: 'development/navigation-menu.md'
-        - Home Page Panels: 'development/homepage.md'
-        - User Preferences: 'development/user-preferences.md'
-        - Release Checklist: 'development/release-checklist.md'
-    - Release Notes:
-        - Version 1.0: 'release-notes/version-1.0.md'
-        - Version 1.1: 'release-notes/version-1.1.md'
-        - Version 1.2: 'release-notes/version-1.2.md'
-        - Version 1.3: 'release-notes/version-1.3.md'
-        - Version 1.4: 'release-notes/version-1.4.md'
+  - Overview: "index.md"
+  - Documentation:
```

Comment:
I recognize that this is mostly just porting over the existing documentation structure as-is, but while we're touching this, I'd be tempted to make the following changes:

- Move `docker/index.md` into the `Installation` heading rather than having a separate top-level heading with only one item under it.
- Move `Administration` up to between `External Authentication` and `User Guides`
- Combine `User Guides` and `Getting Started in the Web UI` into a single `User Guides` section, starting with the `Getting Started` pages and ending with the `User Guides` pages
- Make `Plugins` the last item under `Documentation` instead of having it come before `REST API` and `GraphQL`.

Reply:
Yeah the goal of this first PR is to port the docs without any structural changes or major updates to content, mostly to prevent any possible breakage. The following step after the PR is merged will be to swap domain over to `docs.nautobot.com` so I'd like to reduce the amount of potential broken links at this stage. I'll update this in the PR summary, thanks for the reminder.

Changes to the nav (like the suggested move of the developing plugins bit earlier) should be fine.

Reply:
Nav updated with suggestions.